gw_eccentricity/load_data.py

```diff
@@ -1019,7 +1019,7 @@ def check_sxs_data_dir(origin, **kwargs):
         f"file named `Strain_N{kwargs['extrap_order']}` since `extrap_order` "
         f"is {kwargs['extrap_order']}."}
     if any([kwargs["include_zero_ecc"], kwargs["include_params_dict"],
-            not kwargs["keep_memory"]]):
+            not kwargs.get("keep_memory", True)]):
```

Comment:
Hmm, are we keeping memory by default? Maybe better to do the opposite?

Reply:
No, the default value is `False`. So by default `not kwargs.get("keep_memory", True)])` will be True and therefore one requires to give path to metadata. The default value (the secoond arg `True`) to `kwargs.get()` is to ensure that if someone is loading old format sxs waveform where "keep_memory" is not a kwarg, `not kwargs.get()` will evaluate to False, meaning no requirement to provide metadata file.